extensions/ql-vscode/package.json

```diff
@@ -209,6 +209,24 @@
           "default": null,
           "description": "Path to a directory where the CodeQL extension should store query server logs. If empty, the extension stores logs in a temporary workspace folder and deletes the contents after each run."
         },
+        "codeQL.runningQueries.structuredEvaluatorLog": {
+          "type": [
+            "string",
+            null
+          ],
+          "default": "structured-evaluator-log.json",
+          "description": "Path to a file where the evaluator should store structured evaluator logs. If empty, the query server stores the logs in a default location and overwrites them after each query server run."
+        },
+        "codeQL.runningQueries.structuredEvaluatorLog.minify": {
+          "type": "boolean",
+          "default": false,
+          "description": "Minifies the JSON output of structured evaluator logs, removing all whitespace except a single newline between event objects. If not passed, logs are not minified."
```

Comment:
Is there ever a reason for this to be minified? Ie- do we need this option at all?

Reply:
I'm not entirely sure why a user would flip this option on — perhaps only if the file is expected to be massive? I don't know if that's a use case we really expect here. cc @edoardopirovano

Reply:
Hmm, I guess this depends entirely on how the logs are being consumed. In my mind, the plan was that the extension would consume and manage these logs itself and not directly expose them to the user in any way. If this is the case, then in fact there's no reason for the logs to _not_ be minimised, since no human will ever be looking at them so we may as well save a bit of disk space. The disk-space savings are non-negligible, e.g. for a standard suite of queries on a large-ish database:

![image](https://user-images.githubusercontent.com/6748066/154443698-dd7c75c2-4b9b-48fd-ae8e-23634be0ac39.png)

If, however, there's any plans for the logs to be viewed directly by a user, then it makes sense to have them not be minified. Also, I guess having them unminified is helpful for _us_ when debugging things.

Reply:
Thanks for sharing the graph. Hm 🤔 I think it's possible that a user might want to pull the logfile themselves — perhaps they would want to `grep` the log themselves, or something else that our features do not yet support. 

Though, can a user view other existing files in the standard VSCode query run artifacts?)